README.md

```diff
@@ -304,8 +304,30 @@ sudo brew install libpcap
 
 ### Bootstrapping
 
-- MiNiFi C++ offers a bootstrap script in the root of our github repo that will bootstrap the cmake and build process for you without the need to install dependencies yourself. To use this
-  process, please run the command `bootstrap.sh` from the root of the MiNiFi C++ source tree.
+MiNiFi C++ offers bootstrap scripts that will bootstrap the cmake and build process for you without the need to install dependencies yourself.
+
+#### Python based bootstrapping
+
+Set up a virtual environment in the bootstrap folder.
+```
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+After that you can run the bootstrap script that will guide you through the configuration, dependency installation and build processes.
```

Comment:
```suggestion
After that you can run the bootstrap script: `python main.py`. It will guide you through the configuration, dependency installation and build processes.
```

Reply:
I've added wrapper scripts, so I've rephrased this section. https://github.com/apache/nifi-minifi-cpp/pull/1681/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R309